examples/text_classification/pretrained_models/deploy/python/predict.py

```diff
@@ -20,96 +20,29 @@
 import paddlenlp as ppnlp
 from scipy.special import softmax
 from paddle import inference
-from paddlenlp.data import Stack, Tuple, Pad
+from paddlenlp.data import Tuple, Pad
 from paddlenlp.datasets import load_dataset
 from paddlenlp.utils.log import logger
 
+from ...utils import convert_example
```

Comment:
from ...utils import convert_example -> from utils import convert_example 

Because we run the script from the parent directory by
`python deploy/python/predict.py --model_dir=./export`

Reply:
Done. If we do nothing, then thre would be 
> ImportError: No module named utils

I also added the command to set PYTHONPATH in the README.